### DISCUSSION TITLE
Window Defender Antivirus detected oh-my-posh as Trojan

### DISCUSSION THREAD

**Discussion by dodok8** [JanDeDobbeleer/oh-my-posh#1409] [General]
OS : Windows 11 Education 10.0.22000 build 220000
Language: Korean
Powershell Version: 7.2.0 (Installed via Microsoft Store)
Terminal Emulator: Windows Terminal
oh-my-posh version: 'oh-my-posh' (6.29.2)  (Installed via scoop)
oh-my-posh theme: honukai

Window Defender Antivirus detected oh-my-posh as `Trojan:Win32/Wacatac.B!ml` and prevent it. So `Import-Module oh-my-posh` is failed.

![image](https://user-images.githubusercontent.com/17683473/145680069-1f41655c-b878-4d45-9221-0842f01c3af5.png)

```console
Write-Error: C:\Users\dodok\.config\powershell\user_profile.ps1:3
Line |
   3 |  Import-Module oh-my-posh
     |  ~~~~~~~~~~~~~~~~~~~~~~~~
     | Oh My Posh is unable to download and store the latest version. In case you installed using AllUsers
     | and are a non-admin user, please run the following command as an administrator:  Import-Module
     | oh-my-posh

Set-PoshPrompt: C:\Users\dodok\.config\powershell\user_profile.ps1:5
Line |
   5 |  Set-PoshPrompt honukai
     |  ~~~~~~~~~~~~~~
     | The term 'Set-PoshPrompt' is not recognized as a name of a cmdlet, function, script file, or
     | executable program. Check the spelling of the name, or if a path was included, verify that the path is
     | correct and try again.

Loading personal and system profiles took 2779ms.
C:\Users\dodok> Write-PoshDebug
Write-PoshDebug: The term 'Write-PoshDebug' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

I think it's false positive because just two of vaccine in  VirusTotal says `oh-my-posh.exe` is dangerous. [VirusTotal Report](https://www.virustotal.com/gui/file/d02f13218504d2e6ebcc2cb8c6ca7efffc47d2e93560abd32e98bf8967976226?nocache=1)

**Comment by JanDeDobbeleer** [comment]
That's a known false positive. You can also always verify the hash and/or signature if needed.

**Reply by dodok8** [reply]
I got it. Thank you for your answer. I will add it to the whitelist.
